docs/run-oob-locally.md

```diff
@@ -0,0 +1,154 @@
+# OOB update locally on MacOS
+
+## Kafka
+
+### Install kafka: 
+```
+brew install kafka && brew install zookeeper
+```
+### Start kafka and zookeeper servers:
+```
+mkdir ~/kafka && \
+cd ~/kafka && \
+curl https://archive.apache.org/dist/kafka/0.11.0.0/kafka_2.11-0.11.0.0.tgz | tar xvz && \
+sed 's/zookeeper.connect=.*/zookeeper.connect=localhost:2181\/backbeat/' \
+kafka_2.11-0.11.0.0/config/server.properties > \
+kafka_2.11-0.11.0.0/config/server.properties.backbeat
+zookeeper-server-start kafka_2.11-0.11.0.0/config/zookeeper.properties
+kafka-server-start kafka_2.11-0.11.0.0/config/server.properties.backbeat
+```
+
+### Watch and clean ingestion state (Optional):
+```
+zkCli -server localhost:2181/backbeat
+ls /backbeat/ingestion/state
+delete /backbeat/ingestion/state/<location>
+```
+
+## Mongo DB: deploy replica set
+
+CloudServer uses Mongo DB as a metadata backend.
+Vault is also using Mongo DB as a backend.
+
+From: https://docs.mongodb.com/manual/tutorial/deploy-replica-set/
+
+Create the necessary data directories for each member by issuing a command similar to the following:
+
+```
+mkdir -p /tmp/mongodb/rs0-0 /tmp/mongodb/rs0-1 /tmp/mongodb/rs0-2
+```
+
+Start your mongod instances in their own shell windows by issuing the following commands:
+```
+mongod --replSet rs0 --port 27018 --bind_ip localhost --dbpath /tmp/mongodb/rs0-0 --smallfiles --oplogSize 128
```

Comment:
I got an error using `--smallfiles` but could be my mongo version as I have 4.4

Reply:
oh might be I was using v3.6.20

Reply:
I think we should mimic production as much as we can..so match the version from production.